### ISSUE TITLE
Unable to modify proxy error message despite successful modification

### ISSUE THREAD

**Issue by JesseEstum** [chaostoolkit-incubator/chaostoolkit-toxiproxy#12]
I installed this chaostoolkit extension today.  I run an experiment to call function `enable_proxy` and `disable_proxy` which reports the following error:
```text
[2019-08-16 13:02:24 ERROR] Unable to modify proxy MY_PROXY_NAME
[2019-08-16 13:02:24 ERROR]   => failed: AssertionError
```
However, I can verify with `toxiproxy-cli` that the `disable_proxy` action does toggle the proxy to disabled status.

This is was encountered with toxiproxy version 2.1.4.

**Comment by anthu** [comment]
There is an outdated version pushed to PyPi.

This fix never got released:
https://github.com/chaostoolkit-incubator/chaostoolkit-toxiproxy/commit/2480f274f3ed12bdaa511275af0c8007c8e7855c#diff-bd3a8d6ef2b9430e8b201e1e333511ac

**Comment by Lawouach** [comment]
Hi @anthu, I'm confused. Did we miss to cut a release?

**Comment by anthu** [comment]
Yes, looks like this - comparing my local code with the one on GitHub.

**Comment by Lawouach** [comment]
Could this help? chaostoolkit-toxiproxy 0.2.1

**Comment by anthu** [comment]
Amazing - that was fast! Working for me.
